bindings/core/src/method/wallet.rs

```diff
@@ -438,6 +445,13 @@ pub enum WalletMethod {
         #[serde(default)]
         options: Option<TransactionOptions>,
     },
+    /// Send mana.
+    /// Expected response: [`SentTransaction`](crate::Response::SentTransaction)
+    SendMana {
```

Comment:
We don't actually have any other send methods because the bindings just calls `.send()` on them or whatever